README.md

```diff
@@ -1,3 +1,13 @@
+#### This repo is a fork from [cinnamon-pomodoro](https://github.com/gfreeau/cinnamon-pomodoro "Cinnamon Pomodoro")
+### Addet Functions to this Pomodoro Timer
+- Translate this Applet in German
+- Addet GitHub and BitBucked issues
+- Addet popular Task- and ToDo-Lists
+    - Taskwarrior
+    - ToDoist
+    - Asana
+    - Trello
+---
 # Pomodoro Timer for Cinnamon 2.x, 1.8.x
```

Comment:
Lines from 1 to 10 are not ok and should be removed. If you want to add popular tasks and to do lists, it must done somewhere else in the document.